### PR TITLE
Implemented side navbar

### PR DESCRIPTION
**Pull request by richardzeng1**
-Implemented side navbar visible in all files.
-Renamed page names to more accurately reflect pages.

### CODE REVIEW

**Comment by richardzeng1** [comment]
fixes #41

File: src/views/components/navbar.vue [hunk 1]
```diff
@@ -0,0 +1,60 @@
...
+
+</style>
+
+<template>
+
+<div id="sidenav">
+    <div class="sidenav">
+        <a id="sidenav-header" href = "#">TA Feedback System</a>
+        <hr/>
+        <a href = "#/EditCourses">Edit Courses</a>
```
Comment (siefkenj) [inline src/views/components/navbar.vue]: Which views is this navbar for? It is unclear what "Edit Courses" means...

File: src/views/components/navbar.vue [hunk 2]
```diff
@@ -0,0 +1,60 @@
...
+        <a href = "#/QuestionSelect">Select Survey Questions</a>
```
Comment (siefkenj) [inline src/views/components/navbar.vue]: I think "Courses & Sections", "Surveys", "Results", and "Settings" would be better navigation headings.

File: src/views/components/navbar.vue (hunk 1 above)
Comment (F-Ting) [reply src/views/components/navbar.vue]: Does the side have to be attached to a view? I thought it would be consistent across all views?

Comment (F-Ting) [reply src/views/components/navbar.vue]: Oops, I see what you mean now.

File: src/views/components/navbar.vue [hunk 3]
```diff
@@ -0,0 +1,59 @@
...
+
+</style>
+
+<template>
+
+<div id="sidenav">
+    <div class="sidenav">
+        <a id="sidenav-header" href = "#">TA Feedback System</a>
+        <hr/>
+        <a href = "#/EditCourses">Courses and Sections</a>
```
Comment (siefkenj) [inline src/views/components/navbar.vue]: Rethink the names of these routes.

**Comment by siefkenj** [comment]
I'll merge this now, but it still doesn't look right. The routes look too general, but I guess we'll deal with that when smacks us in the face.

**Pull request merged**